README.md

```diff
@@ -98,6 +100,11 @@ Additionally, @pika/web runs all dependencies through Babel via `@preset/env` to
   },
 ```
 
+### CLI
+
+| Option    | Description                                                                                    | Example                          |
+|-----------|------------------------------------------------------------------------------------------------|----------------------------------|
+| `--entry` | Auto-detect `import`s from your codebase. Supports [glob](https://www.npmjs.com/package/glob). | `npx @pika/web --entry 'src/**/*.js'` |
```

Comment:
Just wanted to call this out: I found the surrounding strings to matter.

`--entry src/**/*.js` for some reason seemed to only match the first file it found. In other words, it happened before @pika/web somehow (or in Yargs?). Kinda confused.

`--entry 'src/**/*.js'` is what most want—this passes the full glob to @pika/web for resolution.

Would love any knowledge on this if you have any 🤷‍♂

Reply:
Ah, I think most shells will actually expand `src/**/*.js` themselves. Wrapping it in quotes makes sure that @pika/web gets the actual glob.

https://www.npmjs.com/package/yargs-parser Looks like there's an `array` argument that you can pass to accept multiple values for a parameter, can you try that out to see if we can support both quoted and unquoted?

Reply:
Hm. Played around with the yargs options but couldn’t find anything that works. I also checked out Babel CLI and Prettier CLI and both of them require quotes for globs like this. No quotes are required when specifying individual files; only during globs. That’s the current behavior.

In other words, `--include file.js` and `--include 'src/**/*.js'` are both valid. `--include src/**/*.js` is still _valid_; it’s just not what people are expecting.

Reply:
okay, that works for now. Thanks for digging into it 👍